relm4/src/factory/collections/vec_deque.rs

```diff
@@ -560,6 +560,8 @@ impl<C: FactoryComponent> FactoryVecDeque<C> {
     pub fn iter(
         &self,
     ) -> impl Iterator<Item = &C> + DoubleEndedIterator + ExactSizeIterator + FusedIterator {
-        self.components.iter().map(|component| component.get())
+        self.components
+            .iter()
+            .map(crate::factory::component_storage::ComponentStorage::get)
```

Comment:
I would be surprised if ComponentStorage wasn't in scope here. I don't think we need to fully qualified input path here.

Reply:
True, fixed it